Xamarin.PropertyEditing.Mac/Controls/Custom/CharTextField.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using AppKit;
+using Foundation;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	public class CharTextField : NSTextField
+	{
+		string cachedValueString;
+		NSText CachedCurrentEditor { get; set; }
+
+		public event EventHandler ValidatedEditingEnded;
+
+		public override CoreGraphics.CGSize IntrinsicContentSize => new CoreGraphics.CGSize (30, 20);
+
+		public CharTextField ()
+		{
+			BackgroundColor = NSColor.Clear;
+			ControlSize = NSControlSize.Small;
+			StringValue = string.Empty;
+			TranslatesAutoresizingMaskIntoConstraints = false;
```

Comment:
Most of these common properties already exist in `PropertyTextField` now, I'm not sure this class is necessary.